_config.yml

```diff
@@ -33,15 +33,15 @@ carpentry: "swc"
 # dc-image:         for Data Carpentry Image Processing
 # swc-inflammation: for Software Carpentry based on the inflammation dataset
 # swc-gapminder:    for Software Carpentry based on the Gapminder dataset
-curriculum: "FIXME"
+curriculum: "swc-inflammation"
```

Comment:
Is it the gapminder or inflammation dataset? I know the workshop earlier this year used the gapminder set.

Reply:
Was going to ask about this specifically

Reply:
Just updated.